config/environments/development.rb

```diff
@@ -50,32 +53,15 @@
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true
 
-  # Debug mode disables concatenation and preprocessing of assets.
-  # This option may cause significant delays in view rendering with a large
-  # number of complex assets.
-  config.assets.debug = true
-
-  # Adds additional error checking when serving assets at runtime.
-  # Checks for improperly declared sprockets dependencies.
-  # Raises helpful error messages.
-  config.assets.raise_runtime_errors = true
-
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
-  # Allow requests for all domains e.g. <app>.dev.gov.uk
-  config.hosts.clear
```

Comment:
Think we still need this for govuk-docker

Reply:
Good catch!

Reply:
Have moved to the bottom, where the config hosts stuff usually is.